ct.yml

```diff
@@ -0,0 +1,6 @@
+# chart-repos:
+#   - cassandra=https://charts.helm.sh/incubator
+#   - prometheus=https://prometheus-community.github.io/helm-charts
+#   - elasticsearch=https://helm.elastic.co
+#   - grafana=https://grafana.github.io/helm-charts 
+validate-maintainers: false
```

Comment:
please fix new line at the end of the file

Reply:
Fixed with https://github.com/temporalio/helm-charts/pull/352/commits/5190b68e1972ada4cdb405e5f0bfbd03f801e0ca